.github/workflows/bie-kafka-end2end-test.yml

```diff
@@ -136,7 +137,7 @@ jobs:
             FROM
               claims.bie_contention_event
             WHERE
-              event_details='Lorem ipsum'
+              event_type='${{env.EVENT_TYPE}}'
```

Comment:
[CI: BIE Kafka End-2-End Test](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/5895187894) failed

Reply:
What's the event_type being used for this integration test? It's a bit abstract in which how the query is selected. I would have preferred all to be in a single test file.

Reply:
You added `EVENT_TYPE` and `event_type`. They're not used elsewhere.

See https://github.com/department-of-veterans-affairs/abd-vro/blob/c51fbb917e0009fbde55d7318a227d5cfddb60b2/svc-bie-kafka/src/integrationTest/java/gov/va/vro/services/bie/BieKafkaApplicationTest.java#L68-L72
and https://github.com/department-of-veterans-affairs/abd-vro/pull/1869/files#diff-59cc2c7b74dd0f68f656e8a2b3914053117bf9b2a6ae60fa10816a77e7a90350L61

Reply:
> In the previous iteration of this test - we were using kcat to send the data to kafka ...

Yes. Since I added SSL, kcat would need to authenticate and I couldn't get that to work. Also, `BieKafkaApplicationTest` uses the SSL configuration (https://github.com/department-of-veterans-affairs/abd-vro/blob/c51fbb917e0009fbde55d7318a227d5cfddb60b2/svc-bie-kafka/src/main/resources/application-integration-test.yaml#L15-L21), similar to our Kafka microservice, so the test also ensures the Spring-based SSL config works.

> Does the Java integration test that's added here actually send the data to the db through the xample workflow?

It sends it to our Kafka microservice -- check the microservice code to see what gets saved to the DB.

Reply:
Here are some options:
   1. Move the `Check for saved DB entry in Postgres` logic (https://github.com/department-of-veterans-affairs/abd-vro/blob/c51fbb917e0009fbde55d7318a227d5cfddb60b2/.github/workflows/bie-kafka-end2end-test.yml#L124) into `BieKafkaApplicationTest`.
   2. Read an env var like `EVENT_BODY` (that is set in `bie-kafka-end2end-test.yml`) and set `BieKafkaApplicationTest.kafkaEventBody` to the var value. Then update `Check for saved DB entry in Postgres` to look for the expected DB entry.

Reply:
https://github.com/department-of-veterans-affairs/abd-vro/pull/1869/commits/ac45d2b42246e0fad4a4cbc1f71daac8d2fe3dc9